papierowyRPG_API/Models/Character.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace papierowyRPG_API.Models
+{
+    public class Character
+    {
+        public int Id { get; set; }
+        public int Id_user { get; set; }
+        public int Id_game { get; set; }
```

Comment:
In EFCore you can use other models for relationships
```suggestion
        public User User { get; set; }
        public Game Game { get; set; }
```

Reply:
Yeah but won't it be necessary to get those models from database when just trying to just take for instance game info?

Reply:
I believe that EFCore in background as Owner/Game puts only id of those models and after we are referring to Owner/Game field from Character model its joined